system-test/system-test-script.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+#Stopping the script if any command fails
+set -euo pipefail
+export myvar="system-test/logfile"
+cd ..
+ echo "Starting collection roller....."
+bin/collection-roller 2>&1 > $myvar &
+#Getting the process id of collection roller
+pid1=$!
+#Checking if the collection roller service has started
+if [ -z "$pid1" ]
+then
+      echo "Collection Roller is not running"
+else
+      echo "Collection Roller is running....."
+fi
+echo "Starting alert engine....."
+bin/alert-engine 2>&1 > $myvar &
```

Comment:
Write each services log into its own file, this will overwrite all previous logs by using the `>` operator

Reply:
fixed.